tools/reviewer/job/tasks/ReviewerMetadataUpdaterTask.java

```diff
@@ -153,5 +163,148 @@ public void run() {
   public Boolean shouldRun() {
     return !lock.isLocked();
   }
+
+  public ReviewerConfig getReviewerConfig(String filePath) throws IOException {
+    ReviewerConfig reviewerConfig =
```

Comment:
I think you can avoid creating `reviewerConfig` variable.
I mean you can return the result without creating the variable:
```
return (ReviewerConfig) fileUtils.readPrototxt(filePath, ReviewerConfig.newBuilder());
```

Also in `getStartupOsReviewerConfigPath()` and `getHasadnaReviewerConfigPath()` methods.

Reply:
Done.